build.gradle

```diff
@@ -159,6 +159,15 @@ python {
     envPath = '.venv'
     minPythonVersion = '3.10' // should be 3.10 for local development
 
+    // Amazon Linux support.
+    try {
+        if ("python3.11 --version".execute().waitFor() == 0) {
+            // python3.11 definitely exists at this point, use it instead of 'python3'.
+            pythonBinary "python3.11"
```

Comment:
❓why cant we bind `python` to `python3.11` at the container level?

Reply:
https://github.com/amazonlinux/amazon-linux-2023/issues/459